### PR TITLE
media-video/libva-utils: new version 2.11.1

### PR DESCRIPTION
**Pull request by Karlson2k**
Added new libva-utils version 2.11.1.
Added USE flags to control which tools are built and installed as most people don't want to install any compiled examples and need only vainfo tool.

Requires PR #20227 to be merged.

### CODE REVIEW

**Comment by Karlson2k** [comment]
@aballier @thesamesam Could you, please, check it and linked PRs?

File: media-video/libva-utils/metadata.xml [hunk 1]
```diff
@@ -7,6 +7,10 @@
   </maintainer>
```
Comment (thesamesam) [inline media-video/libva-utils/metadata.xml:7]: Add yourself! Get CCed on bugs!

File: media-video/libva-utils/libva-utils-9999.ebuild [hunk 2]
```diff
@@ -57,5 +72,19 @@ src_configure() {
...
+	else
+		myeconfargs+=( --disable-tests )
+	fi
 	econf "${myeconfargs[@]}"
 }
+
+src_install() {
+	default
+	if ! use test_va_api ; then
+		if [ -e "${ED}"/usr/bin/test_va_api ] ; then
```
Comment (thesamesam) [inline media-video/libva-utils/libva-utils-9999.ebuild:86]: Switched to Bash tests.

File: media-video/libva-utils/libva-utils-9999.ebuild [hunk 3]
```diff
@@ -47,6 +51,17 @@ DOCS=( NEWS CONTRIBUTING.md README.md )
 src_prepare() {
 	default
 	sed -e 's/-Werror//' -i test/Makefile.am || die
+
+	if ! use examples ; then
+		sed -E -e '/^SUBDIRS \+?=/s!( decode\>| encode\>| videoprocess\>| vendor/intel\>| vendor/intel/sfcsample\>)!!g' -i Makefile.am
```
Comment (thesamesam) [inline media-video/libva-utils/libva-utils-9999.ebuild:56]: Missing || dies, adding for you

File: media-video/libva-utils/metadata.xml (hunk 1 above)
Comment (Karlson2k) [reply media-video/libva-utils/metadata.xml:7]: OK, next time

File: media-video/libva-utils/libva-utils-9999.ebuild (hunk 2 above)
Comment (Karlson2k) [reply media-video/libva-utils/libva-utils-9999.ebuild:86]: OK

File: media-video/libva-utils/libva-utils-9999.ebuild (hunk 3 above)
Comment (Karlson2k) [reply media-video/libva-utils/libva-utils-9999.ebuild:56]: Thanks!

**Pull request closed without merging**